### PR TITLE
Updated a few more menu sublabels

### PR DESCRIPTION
**Pull request by micochops**
Hey all!

Ran through the text a couple more times earlier today.
Seems like I missed a few ones.

Sending updates again!

Thanks!

### CODE REVIEW

File: intl/msg_hash_us.h [hunk 1]
```diff
@@ -2733,22 +2733,22 @@ MSG_HASH(MENU_ENUM_SUBLABEL_TAKE_SCREENSHOT,
...
-      "Closes the current game and application. Any unsaved changes might be lost."
+      "Closes the current content. Any unsaved changes might be lost."
       )
 #else
 MSG_HASH(
       MENU_ENUM_SUBLABEL_CLOSE_CONTENT,
-      "Closes the current game. Any unsaved changes might be lost."
+      "Closes the current content. Any unsaved changes might be lost."
       )
 #endif
```
Comment (tony971) [inline intl/msg_hash_us.h]: If both labels are the same, does the ifdef need to exist?

Comment (tony971) [inline intl/msg_hash_us.h]: If both labels are the same, does the ifdef need to exist?

Comment (Alcaro) [inline intl/msg_hash_us.h]: Not squashing them makes it easier to find and review the real changes.

But yes, should be merged.

**Comment by pepedopolous** [comment]
Just one thing: I think 'slowmotion' should be two words 'slow motion'.

P

File: intl/msg_hash_us.h (hunk 1 above)
Comment (micochops) [inline intl/msg_hash_us.h]: should the ifdef be removed? i can alter the PR for submission again. :)

**Pull request merged**